src/Core/src/Platform/iOS/MauiView.cs

```diff
@@ -105,6 +132,25 @@ public override CGSize SizeThatFits(CGSize size)
 
 			CacheMeasureConstraints(widthConstraint, heightConstraint);
 
+			// If for some reason the upstream measure passes in a negative contraint
+			// Lets just bypass this code
+			if (RespondsToSafeArea() && widthConstraint >= 0 && heightConstraint >= 0)
+			{
+				// During the LayoutSubViews pass, we adjust the Bounds of this view for the safe area and then pass the adjusted result to CrossPlatformArrange.
+				// The CrossPlatformMeasure call does not include the safe area, so we need to add it here to ensure the returned size is correct.
+				//
+				// For example, if this is a layout with an Entry of height 20, CrossPlatformMeasure will return a height of 20.
+				// This means the bounds will be set to a height of 20, causing AdjustForSafeArea(Bounds) to return a negative bounds once it has
+				// subtracted the safe area insets. Therefore, we need to add the safe area insets to the CrossPlatformMeasure result to ensure correct arrangement.
+				var widthSafeAreaOffset = SafeAreaInsets.Left + SafeAreaInsets.Right;
+				var heightSafeAreaOffset = SafeAreaInsets.Top + SafeAreaInsets.Bottom;
+
+				var width = double.Clamp(crossPlatformSize.Width + widthSafeAreaOffset, 0, widthConstraint);
+				var height = double.Clamp(crossPlatformSize.Height + heightSafeAreaOffset, 0, heightConstraint);
+
+				return new CGSize(width, height);
```

Comment:
There are quite a few tests failing on iOS due to content scrolling:
<img width="352" alt="image" src="https://github.com/user-attachments/assets/9e339475-bab8-4af1-b28d-d7a4e48748cd" />
Example, FontFamilyLoadsDynamically:
![image](https://github.com/user-attachments/assets/b47eb27e-e262-40ff-9b31-2e14d60069a3)

Could you review if it is correct?